lib/tasks/gias_update.rake

```diff
@@ -0,0 +1,17 @@
+desc "Upsert GIAS data into GiasSchool"
+task gias_update: :environment do
+  logger = Logger.new($stdout)
+  today = Time.zone.today.strftime("%Y%m%d")
+  gias_filename = "edubasealldata#{today}.csv"
+  csv_path = Rails.root.join("tmp/#{gias_filename}")
+
+  logger.info "Downloading the new gias file for #{Time.zone.today}"
+  Down.download(
+    "#{ENV["GIAS_DOWNLOAD_LINK"]}/#{gias_filename}",
+    destination: csv_path
+  )
+  logger.info "Done!"
+
+  logger.info "Importing data"
+  CsvImports::GiasImport.call(csv_path)
+end
```

Comment:
We should also consider deleting the downloaded file at the end of this task. Otherwise we'll litter the disk with single-use CSV files.

One nice advantage of using the [Tempfile](https://rubyapi.org/o/Tempfile) created by `Down.download` (rather than specifying a location for a File) is that Tempfiles are automatically deleted when the Ruby script finishes. However the [recommended best practice](https://rubyapi.org/o/Tempfile#class-Tempfile-label-Good+practices) is to explicitly close and delete the file anyway.